index.html

```diff
@@ -2975,45 +2975,50 @@ <h3>Zero-Knowledge Proofs</h3>
         </ul>
 
         <p>
-This specification describes a data model that supports zero-knowledge proof
-mechanisms. The examples below highlight how the data model can be used to
-issue, present, and verify zero-knowledge <a>verifiable credentials</a>.
+This specification describes a data model that supports selective disclosure
+with the use of zero-knowledge proof mechanisms. The examples below highlight
+how the data model can be used to issue, present, and verify zero-knowledge
+<a>verifiable credentials</a>.
         </p>
 
         <p>
-To use zero-knowledge <a>verifiable credentials</a> the <a>issuer</a> must
-issue a <a>verifiable credential</a> in a manner that enables the <a>holder</a>
-to present the information to a <a>verifier</a> in a privacy-enhancing manner.
+For a <a>holder</a> to use a zero-knowledge <a>verifiable presentation</a>,
+the <a>issuer</a> must have issued a <a>verifiable credential</a> in a manner
```

Comment:
```suggestion
the <a>issuer</a> MUST have issued a <a>verifiable credential</a> in a manner
```

Reply:
@msporny @brentzundel thoughts on this change? I'm pretty sure it's testable (test if a credential can be derived), but not sure if we want this in here.